simexpal/queuesock.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, serve_socket):
 		self._should_stop = False
 
 	@staticmethod
-	def get_display_name(manifest):
+	def get_run_display_name(manifest):
```

Comment:
This should probably be moved to a different file (launch/common.py? Or a new launch/manifest.py?) such that other launchers can use it as well.

Reply:
Maybe in a different PR though (and then we should also refactor other launchers to use this).